protocol/contracts/beanstalk/init/InitDistribution.sol

```diff
@@ -0,0 +1,49 @@
+/*
+ SPDX-License-Identifier: MIT
+*/
+
+pragma solidity ^0.8.20;
+
+import {AppStorage} from "contracts/beanstalk/AppStorage.sol";
+import {C} from "contracts/C.sol";
+import {LibDiamond} from "contracts/libraries/LibDiamond.sol";
+import {Storage} from "contracts/beanstalk/AppStorage.sol";
+
+// NOTE: Values are arbitrary placeholders.
+
+/**
+ * @author funderbrker
+ * @notice Initializes logistics and shipment routes.
+ */
+contract InitDistribution {
+    AppStorage internal s;
+
+    function init() external {
+        s.shipmentRoutes.push(
+            Storage.ShipmentRoute(
+                address(this),
+                bytes4(keccak256("siloReceive(uint256,bytes)")),
```

Comment:
This works, but personally I prefer using the `x.siloReceive.selector` just because it seems to be more foolproof. Could we implment a `IShipmentPlan` interface (that `Distrubution.sol` or if its decided to make the shipment plan functions placed in a `ShipmentPlanFacet` inherits).

Reply:
fixed here
https://github.com/BeanstalkFarms/Beanstalk/commit/08614d4814895da7c59ddd8c24b0eb489ca783f0#diff-b0451bea09abb2293197eb00e388806cf4949fc7fd2952a7c926693780a93f60R34

Reply:
❤️